lib/archiving.rb

```diff
@@ -27,11 +27,16 @@ def self.archive(date)
           end
         end
       end
+
       puts "Removing archived data from database for #{date}..."
       deliveries.find_each do |delivery|
         delivery.destroy
       end
+
       copy_to_s3(date)
+
+      puts "Removing temp local file #{date}.tar.gz copied to S3..."
+      File.delete("db/archive/#{date}.tar.gz")
```

Comment:
Yep that's real bad. Good catch @henare .

I'll need to add tests for the copy_to_s3 method and the archive method to make sure it does what we want. I find it too scary without tests for the fail cases now that we're deleting stuff.